content/en/security/cloud_security_management/setup/agentless_scanning/quick_start.md

```diff
@@ -14,7 +14,11 @@ further_reading:
 
 Designed for new users, the quick start workflow offers an efficient setup process for Cloud Security Management, enabling immediate monitoring of AWS resources. It uses AWS CloudFormation to automate the configuration, and includes the Cloud Security Management features: Misconfigurations, Identity Risks (CIEM), and Vulnerability Management.
 
-<div class="alert alert-info">This article provides instructions for the new user quick start workflow that uses AWS CloudFormation to set up Agentless Scanning. For existing users who want to add a new AWS account or enable Agentless Scanning on an existing integrated AWS account, see the instructions for <a href="/security/cloud_security_management/setup/agentless_scanning/terraform">Terraform</a> or <a href="/security/cloud_security_management/setup/agentless_scanning/cloudformation">AWS CloudFormation</a>.</div>
+<div class="alert alert-info">This article provides instructions for the new user quick start workflow that uses AWS CloudFormation to set up Agentless Scanning.
+For existing users who want to add a new AWS account or enable Agentless Scanning on an existing integrated AWS account, see the instructions for
+<a href="/security/cloud_security_management/setup/agentless_scanning/terraform">Terraform</a>,
+<a href="/security/cloud_security_management/setup/agentless_scanning/cloudformation">AWS CloudFormation</a>,
+ or <a href="/security/cloud_security_management/setup/agentless_scanning/azure_resource_manager">Azure Resource Manager</a>.</div>
```

Comment:
I created a test org. and it looks like the quick start workflow is only available for AWS right now? If that's correct, then I think we should remove Azure Resource Manager from the note, or reword the note so that it's not specific to AWS.

Reply:
Yes, the quick start workflow is not available on Azure yet. I think the paragraph above is a generic paragraph we tried to reuse in most setup pages, but it can lead to some confusion indeed.